frontend/src/components/Listing/ListingCard.tsx

```diff
@@ -6,17 +6,54 @@ import Typography from '@mui/material/Typography';
 import { Button, CardActionArea, CardActions } from '@mui/material';
 import { capitalize } from '@mui/material/utils';
 import AddShoppingCartIcon from '@mui/icons-material/AddShoppingCart';
+import AddEditIcon from '@mui/icons-material/Edit';
 import 'src/styles/ListingCard.css'
+import {useParams} from "react-router-dom";
+import AddIcon from "@mui/icons-material/Add";
+import ListingGrid from "./ListingGrid";
+import DialogBox from "../Dialog/DialogBox";
+import SellerListingForm from "../Seller/SellerListingForm";
+import IFormInput from "../Seller/SellerListingForm";
 
 export interface IListingCardProps {
     cardName: string,
+    title: string,
     author: string,
     price: string,
     image: string,
-    alt: string
+    alt: string,
+    listingId : string,
+    isbn: string,
+    publisher: string,
+    description: string,
+    inventory: string,
+    releaseDate: string,
 }
 
 export default function ListingCard (props: IListingCardProps) {
+    const [editListingDialog, setEditListingDialog] = React.useState(false);
+
+    const handleEditListing = () => {
+        setEditListingDialog(true);
+
+    };
+
+    let { sellerId } = useParams();
+
+    const closeDialog = () => setEditListingDialog(false);
+
```

Comment:
Might want to change this iForm variable to something more like
previousListingsFormValues

Reply:
Names change